include/qore/intern/QoreOperatorNode.h

```diff
@@ -59,6 +60,14 @@ class QoreOperatorNode : public ParseNode {
       ignoreReturnValueImpl();
    }
 
+   DLLLOCAL virtual bool getInParenthesis() const {
```

Comment:
these functions should not be `virtual`; they are never overridden and are instead based on class member `in_parenthesis` (singular) in the base class which should also actually be called `in_parentheses` (plural) to be grammatically correct